tests/integration/tests/test_external_certificates.py

```diff
@@ -372,3 +379,32 @@ def test_vault_certificates(instances: List[harness.Instance]):
 
     # If we deploy a Pod and it becomes Active, the cluster should be functional.
     check_nginx_pod_runs(instance)
+    delete_nginx_pod(instance)
+
+    # Refresh the PKIs across the cluster.
+    leader_cp_certs = {}
+    create_and_assign_certs(
+        client, itertools.chain(cp_certs.items(), worker_certs.items()), leader_cp_certs
+    )
+    instance.exec(
+        ["k8s", "refresh-certs", "--external-certificates", "-"],
+        input=str.encode(yaml.dump(leader_cp_certs)),
+    )
+
+    new_cp_certs = {}
+    create_and_assign_certs(
+        client, itertools.chain(cp_certs.items(), worker_certs.items()), new_cp_certs
+    )
+    cp_node.exec(
+        ["k8s", "refresh-certs", "--external-certificates", "-"],
+        input=str.encode(yaml.dump(new_cp_certs)),
+    )
+    new_worker_certs = {}
+    create_and_assign_certs(client, worker_certs.items(), new_cp_certs)
```

Comment:
I _think_ this needs to be
```suggestion
    create_and_assign_certs(client, worker_certs.items(), new_worker_certs)
```

Reply:
Good catch! Thanks :)